_guides/guides.md

```diff
@@ -43,6 +43,7 @@ permalink: /guides/
 
 ### Cloud
 
+- [Deploying Native Applications on OpenShift with S2I]({{site.baseurl}}/guides/openshift-s2i-guide)
```

Comment:
Can you move this item as the third item of the list?

Reply:
@cescoffier sure, done.